CONTRIBUTING.md

```diff
@@ -78,7 +84,7 @@ Now that you've identified your contribution, your next step will be to use GitH
 6. Click "Create Branch."
 
 **Open Project File in Text Editor**
-1. Open the text editor of your choice. (We suggest using [Atom](atom.io).)
+1. Open the text editor of your choice. (We suggest [Visual Studio Code](https://code.visualstudio.com/.)
```

Comment:
This gave me pause. Do we really _suggest_ VS Code not even being users of it our selves? Should we be saying something more like "Visual Studio Code is a good, free option."?